typescript/infra/src/config/funding/balances.ts

```diff
@@ -0,0 +1,45 @@
+export enum BalanceThresholdType {
+  RelayerBalance = 'relayerBalance',
+  LowUrgencyKeyFunderBalance = 'lowUrgencyKeyFunderBalance',
+  LowUrgencyEngKeyFunderBalance = 'lowUrgencyEngKeyFunderBalance',
+  HighUrgencyRelayerBalance = 'highUrgencyRelayerBalance',
+}
+
+interface BalanceThresholdConfig {
+  configFileName: string;
+  dailyRelayerBurnMultiplier: number;
+  choiceLabel: string;
+}
+
+export const THRESHOLD_CONFIG_PATH = './config/environments/mainnet3/balances';
+
+const RELAYER_BALANCE_TARGET_DAYS = 8;
+const RELAYER_MIN_DOLLAR_BALANCE_TARGET = 25;
+export const RELAYER_MIN_DOLLAR_BALANCE_PER_DAY =
+  RELAYER_MIN_DOLLAR_BALANCE_TARGET / RELAYER_BALANCE_TARGET_DAYS;
+
+export const balanceThresholdConfigMapping: Record<
+  BalanceThresholdType,
+  BalanceThresholdConfig
+> = {
+  [BalanceThresholdType.RelayerBalance]: {
+    configFileName: 'desiredRelayerBalances.json',
```

Comment:
any reason to have this be different from the rest? wonder if we should just have this type be called `BalanceThresholdType.DesiredRelayerBalance`